solvers/jaxopt.py

```diff
@@ -0,0 +1,89 @@
+import warnings
+
+
+from benchopt import BaseSolver, safe_import_context
+
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    import jax
+    import jaxopt
+    import jax.numpy as jnp
+    import optax
+
+
+@jax.jit
```

Comment:
The tests are failing because when jax is not installed on the system, it is not possible to instanciate this class and get the information. due to the jitting as decorators.